settings.py

```diff
@@ -2256,6 +2256,16 @@ def _determine_couch_databases(couch_databases):
     'test-pna': 'custom.yeksi_naa_reports',
 }
 
+OTHER_DOMAINS_WITH_CUSTOM_MODULES = [
+    'fri',
+    'hope',
+    'logistics',
+    'icds',
```

Comment:
this won't match any domain names but we can remove it because icds is already on the sql backend